ansible/roles/compile-module/tasks/main.yml

```diff
@@ -1,6 +1,6 @@
 ---
 - name: Compile xroad modules {{selected_modules}}
-  command: ../src/compile_module.sh {{modules[item].dir}} {{modules[item].build}}
+  command: ../src/compile_module.sh {{modules[item].dir}} {{modules[item]build}}
```

Comment:
Did yo mean to change this? Looks like accidental find/replace byproduct.

Reply:
You are correct, thanks for the find!